tests/online_identifier/test_online_identifier.py

```diff
@@ -31,8 +31,7 @@ def test_change_own_display_name(main_screen: MainWindow, user_account, new_name
     with step('Go to edit profile settings and change the name of the user'):
         profile_popup.edit_profile().set_name(new_name)
         ChangesDetectedToastMessage().click_save_changes_button()
-        assert ChangesDetectedToastMessage().is_save_changes_button_visible() is False, \
-            f'Save button is not hidden when clicked'
```

Comment:
this `ChangesDetectedToastMessage` is confusing. It is not a really toast right. it is the floating buttons in the bottom of the app.

Reply:
I named it according to symbolic and real name mainWindow_settingsDirtyToastMessage_SettingsDirtyToastMessage = {"container": ":statusDesktop_mainWindow", "id": "settingsDirtyToastMessage", "type": "SettingsDirtyToastMessage", "unnamed": 1, "visible": True} 

Do you want me to rename it somehow?